ANS_JS_template.tex

```diff
@@ -208,88 +220,35 @@ \section{Nuclear Fuel Cycle Simulators}
 
-\begin{itemize}
-\item "Characterize and deploy individual fuel cycle facilities and reactors in a simulation while discretely tracking material movements,
-\item The capability to perform an uncertainty analysis for each element of the fuel cycle and an aggregate uncertainty analysis,
-\item Inclusion of an optimization engine able to optimize simultaneously across multiple objective functions, and
-\item Open and accessible code software and documentation to aid in collaboration between multiple entities and to facilitate software updates."
-\end{itemize}
+One of the reasons why there are so many nuclear fuel cycle simulators is that many of the countries that currently have NPPs have designed their own simulators. While some NFCSs, such as DANESS, are used in multiple countries, many countries have developed their own NFCSs.  DANESS can be seen in figures 2a and 2b as it has been developed and used both in and out of the US. Figure 2b depicts the non-US NFCSs along with some of their software dependencies. Most of the fuel cycle simulators are not open source or available for download; thus, each nation tends to have proprietary software. As discussed with Prof. Kathryn D. Huff, it is also often more efficient, both in terms of time spent as well as better integration, if the different functions are designed by the same team (\cite{redfoot_huff_2016}). Integrating components from one system into a new system can often be more challenging than building the function from scratch. While sharing code can be challenging amongst various program, some calculations or libraries are commonly used by NFCSs. For example, Figure 3 depicts EVOLCODE, FUTURE, and FAMILY21 as all relying on Origen.
+
+The 2016 Electric Power Research Institute report "Program on Technology Innovation: Assessment of Nuclear Fuel Cycle Simulation Tools" takes a commercial software architecture approach to analyze sixteen NFCSs.  The NFCSs are evaluated based on functionality, usability, reliability, performance, and supportability. Of the sixteen NFCSs included, only six appeared to be currently in development: CAFCA, COSI6, CYCLUS, DANESS, ORION, and VISION. As is true with other NFCS Benchmarking papers, the EPRI report used feedback from users of the NFCS \cite{EPRI2016}.  The data in the report is anonymous, making it difficult to distinguish which of the NFCSs particularly stood out for each of the categories. The overall approach of analyzing the software based on commercial software architecture criteria, with many subcriteria, is an important assessment for the adoption of the tool by end users. Some of the criteria the end users involved in the survey viewed as being helpful included: parallel processing, open-source, the flexibility to model complex scenarios, uncertainty/sensitivity analysis, economic capabilities, visual tools, and ability to track many nuclides.
```

Comment:
Just say they are currently in development. "appeared" isn't a good choice of tone here.

Reply:
From the benchmark study, they reached out to 30 or so NFCS and found only six which responded